test/integration/query-interface/changeColumn.test.js

```diff
@@ -235,56 +235,7 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     }
 
     if (dialect === 'sqlite') {
-      it('should not remove unique constraints when adding or modifying columns', async function() {
```

Comment:
What am I missing. This test case doesn'T seem to create a multi column unique index?